src/Apps/MyCollection/Routes/EditArtwork/MyCollectionCreateArtwork.tsx

```diff
@@ -117,12 +116,15 @@ export const MyCollectionCreateArtwork: React.FC<MyCollectionCreateArtworkProps>
       trackSaveCollectedArtwork()
 
       // Store images locally
-
-      if (artworkId && artworkFormImagesRef.current) {
-        await artworkFormImagesRef.current?.saveImagesToLocalStorage(artworkId)
-        await setLocalImagesStoreLastUpdatedAt(
-          IMAGES_LOCAL_STORE_LAST_UPDATED_AT
-        )
+      if (artworkId) {
+        try {
+          await storeArtworkLocalImages(
+            artworkId,
+            localImages.map(({ photoID, ...rest }) => rest)
+          )
```

Comment:
Moved the logic into the `handleSubmit` function because only here the image IDs are available (returned from the mutation).

Reply:
makes sense